app/resources/views/pages/signin.blade.php

```diff
@@ -11,7 +11,8 @@
 			<div class="signin-form">
 				<form method="post" action="/signin">
                     {!! csrf_field() !!}
-					@include('pages.validation_messages', array('errors'=>$errors))					
+					@include('pages.validation_messages', array('errors'=>$errors))
+					<strong>{{ $confirmmessage or '' }}</strong>
```

Comment:
Wrap the strong message in an if like this:
```
@if ( $confirmmessage )
<strong>{{ $confirmmessage }}</strong>
@endif
```
This is pretty minor but it'll keep the HTML in the resulting page just a little cleaner in cases when the confirmmessage is empty.